jest.unit.conf.json

```diff
@@ -4,7 +4,10 @@
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/frontend/test/__mocks__/fileMock.js",
     "^promise-loader\\?global\\!metabase\\/lib\\/ga-metadata$": "<rootDir>/frontend/src/metabase/lib/ga-metadata.js"
   },
-  "testPathIgnorePatterns": ["<rootDir>/frontend/test/.*/.*.tz.unit.spec.js"],
+  "testPathIgnorePatterns": [
+    "<rootDir>/frontend/test/.*/.*.tz.unit.spec.js",
+    "<rootDir>/frontend/test/.*/.*.styled.js*"
```

Comment:
Isn't this to specify the test files and not coverage?

Reply:
Coverage — yes, the draft PR title was misleading, apologies for that.

Reply:
I still don't follow, is it necessary to add a new entry to `testPathIgnorePatterns`?

Reply:
No, not necessary at all, my mixup from inattentive experimentation.

Removed.